src/amqp_channel.erl

```diff
@@ -215,6 +215,8 @@ next_publish_seqno(Channel) ->
 %% @doc Wait until all messages published since the last call have
 %% been either ack'd or nack'd by the broker.  Note, when called on a
 %% non-Confirm channel, waitForConfirms returns an error.
+%% @param Channel: the channel on which to wait.
+%% @end
 wait_for_confirms(Channel) ->
     wait_for_confirms(Channel, amqp_util:call_timeout()).
```

Comment:
We can't use the value returned by `amqp_util:call_timeout()` because it's in milliseconds. Also, it is an unrelated parameter, I don't know why it was used in the first place.

So far, the default was 60000 seconds (almost 17 hours). I think we can safely replace it by `infinity`.

Reply:
The idea was to use 60 seconds as the default, not `infinity`.

Reply:
But it would be a change of behavior: people might get timeouts where they didn't before.

Reply:
If something takes more than 60 seconds and times out before, maybe it should be visible. The decision to use 60 second timeouts in channel operations is not new. We have tried 5 and 15 seconds before, with 5 producing a lot of false positives, 15 produces much fewer. So we have agreed on 60.

Reply:
I'll fix this! Thanks for noticing @dumbbell